libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Login/LoginHost/NewLoginHostView.swift

```diff
@@ -80,7 +80,7 @@ struct NewLoginHostView: View {
         }
     }
     
-    func save() {
+    func save(host: String, hostLabel: String) {
```

Comment:
This is to make it testable, could also move it into a view model but this is the only logic in the view so I was on the fence if it was worth it